### ISSUE TITLE
cssreset.css HTML5 tags support

### ISSUE THREAD

**Issue by danielsokolowski** [yui/yui3#1989]
Hello Epic YUI Team,

Any thoughts on adding HTML5 support to the reset style sheet? My reaching out is due to the way '<main>' tag is being rendered as inline in IE8-11 and the simple fix to style is as 'display: block' which would make sense to be added to the reset sheet.

Thoughts?

**Comment by apm** [comment]
https://github.com/yahoo/pure/ - this is the successor to the css solutions in YUI.  Any new support in reset would happen there.

**Comment by danielsokolowski** [comment]
@apm thanks for the response, pure however does not seem to have reset just a normalize - am I correct on this?

**Comment by apm** [comment]
You're right - it is built on http://necolas.github.io/normalize.css/, so you probably need to move to that.  That or fork this and modify it to suit your purposes.  There isn't a YUI team any longer, so cssreset won't be updated.
